app/javascript/components/widgets/twitter/panel.jsx

```diff
@@ -87,7 +90,12 @@ Twitter.propTypes = {
         retweetCount: PropTypes.number.isRequired,
       }).isRequired,
       media: PropTypes.shape({
-        images: PropTypes.arrayOf(PropTypes.string),
+        images: PropTypes.arrayOf(
+          PropTypes.shape({
+            id: PropTypes.number.isRequired,
+            mediaUrl: PropTypes.string.isRequired,
+          }),
+        ),
```

Comment:
Does PropTypes.shape enforce what each element of the array is supposed to look like?